components/modal/stories/modal.stories.js

```diff
@@ -3,23 +3,37 @@ import { Template } from "./template";
 
 export default {
   title: "Modal",
-  description: "The Modal component is...",
+  description: "A modal component used primarily by Dialog.",
   component: "Modal",
   argTypes: {
-    size: {
-      name: "Size",
-      type: { name: "string", required: true },
+    isOpen: {
+      description: "Whether the modal is open (visible).",
+      defaultValue: true,
+      table: {
+        type: { summary: "boolean" },
+        category: "State",
+        defaultValue: { summary: "true" },
+      },
+      control: "boolean",
+    },
+    variant: {
+      description: "Controls how the modal fills the available space. <ul><li>\"responsive\" will fill the screen on small viewports.</li><li>\"fullscreen\" will fill almost all of the available screen space.</li><li>\"fullscreenTakeover\" will fill all of the available screen space.</li></ul>",
+      defaultValue: undefined,
       table: {
         type: { summary: "string" },
         category: "Component",
+        defaultValue: { summary: "undefined" },
```

Comment:
```suggestion
```

Can you leave off the defaultValues? Since it's a CSS library, nothing is really added automatically - it's all opt-in. You can set default values in the args section below.

Reply:
Sure. Resolved in [fb67da1](https://github.com/adobe/spectrum-css/pull/1655/commits/fb67da1623b9335c38aafbaba085e361e0d5f238)